apps/docs/content/components/River/index.mdx

```diff
@@ -18,14 +18,14 @@ River content should be short and concise, no longer than 3 or 4 sentences and i
   <Do>
     <img
       alt=""
-      src="https://github.com/primer/brand/assets/175638/8b3057f9-00c5-4bf5-af44-00b0129c6c6e"
+      src="https://github.com/primer/brand/assets/912236/a213a475-c5b1-49d0-8c82-5c99597a24ea"
```

Comment:
This one is missing the CTA. Is that intended?

Reply:
not intended 🤦 fixed!